### ISSUE TITLE
0 and False for boolean marked as a change

### ISSUE THREAD

**Issue by jorenvh1** [owen-it/laravel-auditing#216]
Is there a way to change the configuration for validating booleans?
I send False to my DB but it will be stored as 0. Because of this behaviour, it will be marked as a change when it's not.

Thanks in advance

**Comment by quetzyg** [comment]
I'll look into it.

**Comment by jorenvh1** [comment]
I currently fixed it by just replacing 'False' with 0 and 'True' with 1 so it doesn't get logged as a change.
Still interested in how you'd fix this problem though ;)

**Comment by quetzyg** [comment]
Hey @jorenvh1,

This is an issue with Eloquent's `getDirty()` method. I guess we'll just have to wait until it gets a proper fix. 
Meanwhile, you'll have to resort to the workaround you described.

If you're interested, you can also track how things are progressing in the current [open issue](https://github.com/laravel/internals/issues/349) about this problem, which has links [to](https://github.com/laravel/framework/issues/16823) [other](https://github.com/laravel/framework/issues/15060) [closed](https://github.com/laravel/framework/issues/10735) [issues](https://github.com/laravel/framework/issues/8972).

Marking this as closed.

**Comment by LeonAlvarez** [comment]
@quetzyg according to https://github.com/laravel/ideas/issues/349 it should work, but I still facing it on v9 laravel 5.8

**Comment by quetzyg** [comment]
@LeonAlvarez, that has been fixed since version 5.5. If you're still having a similar issue, you're either not on the versions you think you are, or something else is wrong with your code.

See the [troubleshooting](http://www.laravel-auditing.com/docs/9.0/troubleshooting) for a workaround, using [this](https://gist.github.com/crashkonijn/7d581e55770d2379494067d8b0ce0f6d) trait.

**PS:** I'm no longer a collaborator to this package, since I started working on my own auditing package.
